nbconvert/nbconvertapp.py

```diff
@@ -69,6 +70,10 @@ def validate(self, obj, value):
          "(the default behaviour is to abort conversion). This flag "
          "is only relevant if '--execute' was specified, too.")
         ),
+    'stdin' : (
+        {'NbConvertApp' : {'from_stdin' : True}},
```

Comment:
This can include `'writer_class': 'StdoutWriter'`, removing the need for `_from_stdin_changed` below.